libbeat/publisher/broker/membroker/config.go

```diff
@@ -1,9 +1,26 @@
 package membroker
 
+import (
+	"errors"
+	"time"
+)
+
 type config struct {
-	Events int `config:"events" validate:"min=32"`
+	Events         int           `config:"events" validate:"min=32"`
```

Comment:
well, we need to buffer some data for pushing to the outputs. Without queue the pipeline won't be able to operate -> enforce minimum so users don't use 1 or 0.